its/ruling/src/test/java/org/sonarsource/iac/IacRulingTest.java

```diff
@@ -95,6 +95,7 @@ void test_kubernetes() throws IOException {
     Map<String, String> properties = new HashMap<>();
     properties.put("sonar.inclusions", "sources/kubernetes/**/*.yaml, ruling/src/test/resources/sources/kubernetes/**/*.yaml," +
       "sources/kubernetes/**/*.yml, ruling/src/test/resources/sources/kubernetes/**/*.yml");
+    properties.put("sonar.kubernetes.internal.helm.enable", "true");
```

Comment:
I think we should include this for PluginTest as well

Reply:
Enabled, however, it changes nothing